File: prak3_java/Processor.java
public class Processor {
    
    private String name;
    private int price;

    //konstruktor
    public Processor(){
    }

    public Processor(String name, int price){
        this.name = name;
        this.price = price;
    }

    //getter setter
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

}

File: prak3_c++/Processor.cpp
#include <string>

using std::string;

class Processor {
    private:
        string name;
        int price;

    public:
        //konstruktor
        Processor(){ 
        }

        Processor(string n, int p){
            name = n;
            price = p;
        }


        //setter
        void setName(string n){
            name = n;
        }

        void setPrice(int p){
            price = p;
        }

        //getter
        string getName(){
            return name;
        }

        int getPrice(){
            return price;
        }


        //destruktor
        ~Processor(){
        }
};
